modules/test-structure/save_test_data_test.go

```diff
@@ -218,3 +219,23 @@ func TestSaveAndLoadNamedInts(t *testing.T) {
 	assert.Equal(t, expectedData1, actualData1)
 	assert.Equal(t, expectedData2, actualData2)
 }
+
+func TestSaveAndLoadKubectlOptions(t *testing.T) {
+	t.Parallel()
+
+	tmpFolder, err := ioutil.TempDir("", "save-and-load-kubectl-options")
+	if err != nil {
+		t.Fatalf("Failed to create temp dir: %v", err)
+	}
```

Comment:
Can you switch this to using `require.NoError(t, err)`? `require` is imported as `github.com/stretchr/testify/require`. We've been (slowly) transitioning to using `require` instead of `if => t.Fatal`, so for new code we use this pattern.